app/assets/javascripts/discourse/app/components/sidebar/anonymous-categories-section.hbs

```diff
@@ -0,0 +1,20 @@
+<Sidebar::Section
+  @sectionName="categories"
+  @headerLinkText={{i18n "sidebar.sections.categories.header_link_text"}} >
+
+  {{#each this.sectionLinks as |sectionLink|}}
+    <Sidebar::SectionLink
+      @route={{sectionLink.route}}
+      @title={{sectionLink.title}}
+      @content={{sectionLink.text}}
+      @currentWhen={{sectionLink.currentWhen}}
+      @model={{sectionLink.model}}
+      @prefixType={{sectionLink.prefixType}}
+      @prefixValue={{sectionLink.prefixValue}}
+      @prefixColor={{sectionLink.prefixColor}} >
+    </Sidebar::SectionLink>
+  {{/each}}
+  <div class="sidebar-more-section-links-details-content-secondary">
```

Comment:
I don't think there is a need for this class too. We can just treat this as a normal section link and pass the arguments directly. That means no need for the `this.moreLink` object too.